projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -77,7 +77,7 @@ export class TableControlsComponent implements OnChanges {
   public searchPlaceholder?: string = 'Search...';
 
   @Input()
-  public selectMap?: Map<string, SelectFilter>;
+  public selectFilterItems?: SelectFilter[] = [];
```

Comment:
It shouldn't be a problem. The Output from the dropdown is the value, which is now an object keyValue pair of attribute:value so the listener knows which filter was changed.

Reply:
But that's in the caller - it's now on the caller to provide a value that it knows how to disambiguate from other values in other filters, right? I'm fine leaving it as is since the renderer is likely the only place we're using these controls in a complex way, but from a generic component perspective, I think that's a design issue.

(an alternative would be something like `{source: SelectFilter; value: SelectOption<..>}`)